style.css

```diff
@@ -20,6 +18,49 @@
   display: none;
 }
 
+.menuim {
+  margin-right: 40px;
+  height: 0%;
+  width: 0%;
+}
+
+.nav_expand {
+  position: absolute;
+  height: 100%;
+  width: 375px;
+  background-color: #6070ff;
+  opacity: 0.92;
+  display: none;
+}
```

Comment:
- [ ] On some screen sizes the mobile menu is not covering the entire screen. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/174820011-bf8cf3a1-bcd4-4760-924d-f0b15a31197b.png)